### PR TITLE
CHE-1340: Move ide related modules to /che/ide

### PR DESCRIPTION
**Pull request by vparfonov**
move:
core/ide -> ide
core/commons/che-core-commons-gwt -> ide/commons-gwt

Signed-off-by: Vitaly Parfonov vparfonov@codenvy.com

### CODE REVIEW

**Comment by vparfonov** [comment]
@skabashnyuk @gazarenkov @benoitf @vzhukovskii @TylerJewell

**Comment by benoitf** [comment]
2,457 changes ^^

File: ide/che-core-ide-api/pom.xml [hunk 1]
```diff
@@ -18,6 +18,7 @@
         <groupId>org.eclipse.che.core</groupId>
         <version>4.4.0-RC1-SNAPSHOT</version>
     </parent>
+    <groupId>org.eclipse.che.core</groupId>
```
Comment (benoitf) [inline ide/che-core-ide-api/pom.xml:21]: groupId should be org.eclipse.che.ide now as it's no longer in the core part

Comment (benoitf) [inline ide/che-core-ide-api/pom.xml:21]: (could be done in a next commit)
to be sure groupId are aligned with the location of modules

Comment (benoitf) [inline ide/che-core-ide-api/pom.xml:21]: artifact id or module name are still using "core" inside their name as well
che-core-ide-api vs che-ide-api

**Comment by vparfonov** [comment]
I think here we don't nee review.
Just moving directory, any changes inside classes, mention you to be sure you are informed because you not in our office

File: ide/che-core-ide-api/pom.xml (hunk 1 above)
Comment (vparfonov) [inline ide/che-core-ide-api/pom.xml:21]: agree it will be done in other PR

**Comment by vzhukovs** [comment]
Okay, I'll apply the same movement in my branches.

**Comment by skabashnyuk** [comment]
ok

**Pull request merged**